doc/setup/install.rst

```diff
@@ -525,13 +533,27 @@ Dependencies
 ^^^^^^^^^^^^
 
 Odoo dependencies are listed in the `requirements.txt` file located at the root of the Odoo
-community directory. Most of them can be installed with **pip**.
+community directory. Most of them can be installed with **pip** altough some libraries require
+installing other system package too.
 
 .. tip:: It can be preferable to not mix python modules packages between different instances of Odoo
          or with your system. You can use virtualenv_ to create isolated Python environments.
 
-Navigate to the path of your Odoo Community installation (`YourOdooCommunityPath`) and run **pip**
-on the requirements file:
+First, make sure all system dependencies are met:
```

Comment:
Specify for which system this is needed. Ubuntu or Debian? We try to keep general information so it also works on non-apt system (fedora, arch,...)

Reply:
Debian, let me try for ubuntu..